dockers/Screenshotter/screenshotter.js

```diff
@@ -372,16 +377,32 @@ function takeScreenshot(key) {
     }
 
     const url = katexURL + "test/screenshotter/test.html?" + itm.query;
-    driver.get(url);
-    if (opts.wait) {
-        browserSideWait(1000 * opts.wait);
+    driver.call(loadMath);
+
+    function loadMath() {
+        if (!opts.reload && driverReady) {
+            driver.executeAsyncScript(
+                    "var callback = arguments[arguments.length - 1]; " +
```

Comment:
I had to look this up in the selenium docs.  It seems like such an awkward API for interacting with the browser.  Thanks for figuring out all of this stuff.